modelon/impact/client/entities/workspace.py

```diff
@@ -305,27 +386,30 @@ def download(self, options: Dict[str, Any], path: str) -> str:
             f.write(data)
         return ws_path
 
-    def clone(self) -> 'Workspace':
-        """Clones the workspace.
-        Returns a clone Workspace class object.
+    # TODO: Cloning workspace is not implemented on feature branch
+    # def clone(self) -> 'Workspace':
```

Comment:
Would be better to keep this uncommented? This seems a little messy and we probably want the clone method to exist so we should keep it. If we remove methods we will have to do a BREAKING CHANGE commit to get correct versioning.